tests/test_core.py

```diff
@@ -8,23 +8,16 @@
 # more details.
 
 import sys
-
 from unittest import TestCase
 
 from flask import Blueprint, Flask, request, url_for
 
-from flask_menu import (
-    Menu,
-    current_menu,
-    register_menu,
-)
-from flask_menu.classy import (
-    register_flaskview,
-    classy_menu_item,
-)
+from flask_menu import Menu, current_menu, register_menu
+from flask_menu.classy import classy_menu_item, register_flaskview
 
 
 class FlaskTestCase(TestCase):
+
```

Comment:
There should be no newline before class docstings anymore, see: https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/28.